### PR TITLE
Expose types for easier interface implementations.

### PR DESCRIPTION
**Pull request by Gouthamve**
TombstoneReader is exposed but Intervals is not.

Signed-off-by: Goutham Veeramachaneni <cs14btech11014@iith.ac.in>

---
This change is [Reviewable](https://reviewable.io/reviews/prometheus/tsdb/128)

### CODE REVIEW

File: tombstones.go [hunk 1]
```diff
@@ -131,39 +131,40 @@ func readTombstones(dir string) (tombstoneReader, error) {
...
 
-func (t tombstoneReader) add(ref uint32, itv interval) {
+func (t tombstoneReader) add(ref uint32, itv Interval) {
 	t[ref] = t[ref].add(itv)
 }
 
-type interval struct {
+// Interval represents a single time-interval.
+type Interval struct {
 	mint, maxt int64
```
Comment (fabxc) [inline tombstones.go]: Is there a lot of user if we can have an exported TombstoneReader but cannot actually read the values of the intervals it exposes?

**Comment by Gouthamve** [comment]
Updated PTAL.

**Comment by fabxc** [comment]
👍

**Comment by fabxc** [comment]
👍

**Pull request merged**